src/database/database.ts

```diff
@@ -6,7 +6,7 @@ export const createUser = async (user: User): Promise<User> => {
   const data = readDatabase()
 
   if (data.users.find((existingUser) => existingUser.email === user.email))
-    throw new Error('User already exists')
+    console.log('User already exists')
```

Comment:
I would expect the candidate to mention here that this should be as it was and not just a console.log